### ISSUE TITLE
Dates in UI are shown in server time (or UTC?), not in local time of user

### ISSUE THREAD

**Issue by sebix** [SK-CERT/Taranis-NG#24]
My server uses UTC, not the local time zone. The timestamps shown in the UI, e.g. collected and published dates, are therefore shown in UTC, not my own time zone.

**Comment by milankowww** [comment]
You're right. We will modify the code so that the API will always send UTC, and the GUI would convert to local time zone for display.
